eng/Directory.Build.Common.targets

```diff
@@ -188,7 +188,15 @@
   
   <!--TODO: end-->
 
-  <!-- *********** Management Client Library Override section ************* -->
+  <!-- *********** Files needed for LRO ************* -->
+  <ItemGroup Condition="'$(IncludeOperationsSharedSource)' == 'true'">
```

Comment:
Do we need to define this here also?

https://github.com/Azure/azure-sdk-for-net/blob/50f7c35249fb263733d55edd1cf6662ede1fdbea/eng/Directory.Build.Common.props#L76

Reply:
Not all track 2 projects would need these files, I presume, such as those not based on HTTP e.g. Service Bus. Including them would unnecessary inflate package size.

Reply:
I don't think so because the idea is there are no default values you have to opt in

Reply:
I meant define it and default it to false. If not defining it at all is equivalent, that's fine too.